pkgs/applications/version-management/gcli/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+, curl
+, autoreconfHook
+, pkg-config
+, yacc
+, flex
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gcli";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "herrhotzenplotz";
+    repo = "gcli";
+    rev = version;
+    sha256 = "sha256-ry+T39gFVPfHazAbv97UFpMIH1Dbbw6tZwsn9V4uRec=";
```

Comment:
```suggestion
    hash = "sha256-ry+T39gFVPfHazAbv97UFpMIH1Dbbw6tZwsn9V4uRec=";
```

Reply:
Another "new" thing I didn't realize, thank you!